ethpm/_utils/backend.py

```diff
@@ -30,6 +27,12 @@
     RegistryURIBackend,
 )
 
+try:
+    from ipfshttpclient.exceptions import ConnectionError as IpfsConnectionError
```

Comment:
This was renamed to `IpfsConnectionError` because there is a builtin `ConenctionError`... so we'd be mistakenly catching an unwanted exception if the `ipfshttpclient` module is not available.